clinica/pipelines/deeplearning_prepare_data/deeplearning_prepare_data_cli.py

```diff
@@ -0,0 +1,129 @@
+        clinica_comp.add_argument("extract_method",
+                                  help='''Method used to extract features. Three options:
+                                       'slice' to get 2D slices from the MRI,
+                                       'patch' to get 3D volumetric patches or
+                                       'whole' to get the complete MRI.''',
+                                  choices=['slice', 'patch', 'whole'], default='whole'
+                                  )
+
+        # group_id can be used by certain pipelines when some operations are performed at the group level
+        # (for example, generation of a template in pipeline t1-volume)
+        # clinica_comp.add_argument("group_id",
+        #                           help='User-defined identifier for the provided group of subjects.')
+
+        # Add your own pipeline command line arguments here to be used in the
+        # method below. Example below:
+        optional = self._args.add_argument_group(PIPELINE_CATEGORIES['OPTIONAL'])
+
+        optional.add_argument(
+                '-psz', '--patch_size',
+                help='''Patch size (only for 'patch' extraction) e.g: --patch_size 50''',
+                type=int, default=50
+                )
+        optional.add_argument(
+                '-ssz', '--stride_size',
+                help='''Stride size (only for 'patch' extraction) e.g.: --stride_size 50''',
+                type=int, default=50
+                )
+        optional.add_argument(
+                '-sd', '--slice_direction',
+                help='''Slice direction (only for 'slice' extraction). Three options:
+                '0' -> Sagittal plane,
+                '1' -> Coronal plane or
+                '2' -> Axial plane''',
+                type=int, default=0
+                )
+        optional.add_argument(
+                '-sm', '--slice_mode',
+                help='''Slice mode (only for 'slice' extraction). Two options:
+                'original' to save one single channel (intensity),
+                'rgb' to saves three channel (with same intensity).''',
+                choices=['original', 'rgb'], default='rgb'
+                )
+
+        # Clinica standard arguments (e.g. --n_procs)
+        self.add_clinica_standard_arguments()
+
+        # Add advanced arguments
+        advanced = self._args.add_argument_group(PIPELINE_CATEGORIES['ADVANCED'])
+        advanced.add_argument("-aa", "--advanced_arg", default='Advanced',
+                              help='Your advanced argument (default: --advanced_arg %(default)s).')
+
+    def run_command(self, args):
+        """Run the pipeline with defined args."""
+        import os
+        from networkx import Graph
+        from colorama import Fore
+        from .deeplearning_prepare_data_pipeline import DeepLearningPrepareData
+        from clinica.utils.stream import cprint
+        from clinica.utils.ux import print_end_pipeline, print_crash_files_and_exit
```

Comment:
Some imports are not used:
```suggestion
        from networkx import Graph
        from .deeplearning_prepare_data_pipeline import DeepLearningPrepareData
        from clinica.utils.ux import print_end_pipeline, print_crash_files_and_exit
```